### DISCUSSION TITLE
V3.8 Update check seems to be broken

### DISCUSSION THREAD

**Discussion by vmsman** [lxdware/lxd-dashboard#81] [Q&A]
In the "about" menu and "check for updates", it appears to not realize that I have updated.
![image](https://github.com/lxdware/lxd-dashboard/assets/19209494/78050158-4fdc-41b7-83bf-4063418c92b9)

**Comment by matthewalanpenning** [comment]
Thanks. I had tagged the release v.3.8.0 in GitHub, but it should have been v3.8.0. The tag is updated and should be working correctly now. Thanks for the find.
